### ISSUE TITLE
The Repository structure is not as required. 

### ISSUE THREAD

**Issue by pushpdeep** [basitali97/FCFS-Vlab#1]
Please refer this structure here and correct it ASAP.

the repository structure -
http://vlabs.iitb.ac.in/gitlab/Community-Docs/New-Lab-development/Round-templates

Documentation on porting of the experiment -
https://docs.google.com/document/d/1y8M1qslLjtyHmGnjclJQVEscNnB2n2JJTpeIADiuSkk/edit?usp=sharing
@basitali97

**Comment by basitali97** [comment]
Issue is resolved and git repository structure is proper.

**Comment by pushpdeep** [comment]
@basitali97  Pedagogy content is missing

**Comment by basitali97** [comment]
Issue is resolved and git repository structure is proper.

**Comment by pushpdeep** [comment]
@basitali97 yes , but please approach the faculty and get the content for pedagogy and storyboard.
